### PR TITLE
HPy sprint status

### PR DESCRIPTION
**Pull request by fangerer**

### CODE REVIEW

**Comment by fangerer** [comment]
@cfbolz this is the HPy blog post about the sprint report

**Review by hodgestar — APPROVED** [review]

File: posts/2022/09/hpy_sprint_status.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+.. link:
+.. description:
+.. type: text
+-->
+
+As we announced in
+[this blog post](https://hpyproject.org/blog/posts/2022/07/dusseldorf-sprint-2022/),
+we are currently having our joint PyPy/HPy/GraalPy sprint in Dusseldorf.
+
+Yesterday, we did the HPy sprint report. Most notably, we also had several
```
Comment (steve-s) [inline posts/2022/09/hpy_sprint_status.md:17]: I would avoid relative dates

File: posts/2022/09/hpy_sprint_status.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+<iframe width="560" height="315" src="https://www.youtube.com/embed/kl5w8uR6hGA" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+<!--TEASER_END-->
+
+
+## HPy Sprint Call with CPython Core Developers
+
+One of the main goals in the sprint was to write a PEP to officially propose
+HPy as an Python C extension API. We started to do so but before we actually
+submit this PEP, we wanted to have some feedback if that makes sense. So,
```
Comment (steve-s) [inline posts/2022/09/hpy_sprint_status.md:31]: minor suggestion: ~if that makes sense~ and ~and quite~ some of them
joined.

File: posts/2022/09/hpy_sprint_status.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+we invited the Python core developers to our call and quite some of them
+followed the invitation.
+
+We therefore gave a short overview of HPy since not everyone might know it yet.
```
Comment (steve-s) [inline posts/2022/09/hpy_sprint_status.md:35]: minor: We ~therefore~ gave...

File: posts/2022/09/hpy_sprint_status.md [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+We talked about the goals and non-goals of HPy and basically asked if Python
+core developers would be willing to support our project
```
Comment (steve-s) [inline posts/2022/09/hpy_sprint_status.md:37]: participate in HPy design discussions (and endorse HPy as the go-to API for Python extensions(?) -- not sure, does it follow from the discussion?). We discussed PSF endorsement and I think the conclusion was that we need to talk to PSF.

**Pull request merged**